src/components/VitalStat.vue

```diff
@@ -7,121 +7,128 @@
  * @param value {Number} sum of dependency values
  */
 export default {
-  props:{'stat': Object, 'scale': Number, 'value': Number},
-  data(){
-    return{
-      initialValue:(this.stat.defaultValue) ? this.stat.defaultValue : this.finalValue()
-    }
+  props: { stat: Object, scale: Number, value: Number },
+  data() {
+    return {
+      initialValue: this.stat.defaultValue
```

Comment:
ok preco?
`? :` zapis podmienky byva jednoriadkovy, preco to davat na 3 riadky

Reply:
to urobil pretiffy

Reply:
nedalo sa to citat, lebo zasadne nepises medzery, tak nez som sa zacal divat do kodu, tak som pouzil pretify

Reply:
nepisem no...